### PR TITLE
Removed all scss from the spiff folder and copied over all the css-co…

### PR DESCRIPTION
**Pull request by ElishaStaks**
…mpiled files into its own css folder

### CODE REVIEW

File: css/custom.css [hunk 1]
```diff
@@ -1,6 +1,7 @@
 @charset "UTF-8";
```
Comment (stewartmatheson) [inline css/custom.css:1]: is `@import` a scss thing? I have a feeling it's not valid CSS. Can you confirm that.

File: css/nucleus.css [hunk 2]
```diff
@@ -0,0 +1,617 @@
...
+
+button,
+input[type="submit"] {
+  cursor: pointer;
+  user-select: none;
+  vertical-align: middle;
+  white-space: nowrap;
+  border: inherit; }
+
+/*# sourceMappingURL=nucleus.css.map */
```
Comment (stewartmatheson) [inline css/nucleus.css]: we should not need the mapping link

**Review by stewartmatheson — CHANGES_REQUESTED** [review]

File: css/custom.css (hunk 1 above)
Comment (ElishaStaks) [reply css/custom.css:1]: Import url is not a scss thing, however people are saying it takes a hit on the speed at which your site runs so ill use a different method.

File: css/nucleus.css (hunk 2 above)
Comment (ElishaStaks) [reply css/nucleus.css]: Okay ill remove it.

**Pull request merged**